static/css/components/work.less

```diff
@@ -41,6 +41,11 @@ div.editionAbout {
     margin: 10px 0;
   }
 }
+
+.meta .object {
```

Comment:
I think the select you want here is dd not '.meta .object'

This is overly specific and leads to this issue with the goodreads identifier:
![screen shot 2018-12-27 at 1 17 07 pm](https://user-images.githubusercontent.com/148752/50494706-bfaf4a00-09d9-11e9-8b9b-bddf3a0b149e.png)

Reply:
great catch